articles/android-rxjava/index.md

```diff
@@ -0,0 +1,127 @@
+#### RxJava in Android
+ReactiveX has created libraries for almost all programming languages. It has also extended this to frameworks like Android. This has helped bring the functional reactive paradigm to the languages and frameworks we use. In Android, it helps bring efficiency in application performance. It also ensures a clean and maintainable code. This article will go through the basics of RxJava and how to use it in Android development.
```

Comment:
I'd suggest you use "such us" rather than "like" on you second sentence. 
"This has brought functional reactive..." flows better.

Reply:
done.